pkg/apis/kubeflow.org/v1/pytorch_defaults_test.go

```diff
@@ -152,3 +152,25 @@ func TestSetElasticPolicy(t *testing.T) {
 		})
 	}
 }
+
+func TestSetDefaultNprocPerNode(t *testing.T) {
```

Comment:
I think that this test doesn't verify some edge cases.
For example, the current case can not verify that `.spec.nprocPerNode` isn't overridden when `elasticPolicy` is nil and `.spec.elasticPolicy` isn't nil.

So should we add more test cases?

Reply:
Done